tf-psa-crypto/drivers/builtin/include/mbedtls/ecp.h

```diff
@@ -345,6 +345,8 @@ mbedtls_ecp_group;
 
 #define MBEDTLS_ECP_MAX_BYTES    ((MBEDTLS_ECP_MAX_BITS + 7) / 8)
 #define MBEDTLS_ECP_MAX_PT_LEN   (2 * MBEDTLS_ECP_MAX_BYTES + 1)
+#define MBEDTLS_ECP_MAX_MPI      ((MBEDTLS_ECP_MAX_BYTES + sizeof(mbedtls_mpi_uint)) & \
+                                  ~(sizeof(mbedtls_mpi_uint)-1))
```

Comment:
One more thing: I think it would be better to follow the same method as in `MBEDTLS_ECP_MAX_BYTES`. As far as I can tell they do the same, but using the same method would make it easier to read:
```suggestion
#define MBEDTLS_ECP_MAX_MPI      ((MBEDTLS_ECP_MAX_BYTES + sizeof(mbedtls_mpi_uint) - 1) / \
                                  sizeof(mbedtls_mpi_uint))
```

Reply:
It's not exactly the same `MBEDTLS_ECP_MAX_MPI` is still in bytes just aligned to the nearest 8 (or sizeof(mbedtls_mpi_uint)) if we wanted to do it in the same way it will have to be:
`#define MBEDTLS_ECP_MAX_MPI      ((MBEDTLS_ECP_MAX_BYTES + sizeof(mbedtls_mpi_uint) - 1) / \
                                  sizeof(mbedtls_mpi_uint)) * sizeof(mbedtls_mpi_uint)`
Do you want me it change it to this?
I can also rename it to `MBEDTLS_ECP_MAX_MPI_BYTES` if it makes it clearer?

Reply:
Oh, indeed. Then maybe we could try something like `CHARS_TO_LIMBS(MBEDTLS_ECP_MAX_BYTES) * sizeof(mbedtls_mpi_uint)` or `CHARS_TO_LIMBS(MBEDTLS_ECP_MAX_BYTES) * ciL` or maybe `BITS_TO_LIMBS(MBEDTLS_ECP_MAX_BITS) * sizeof(mbedtls_mpi_uint)` or `BITS_TO_LIMBS(MBEDTLS_ECP_MAX_BITS) * ciL`.

Any of these should do the job and would help readability. (As far as I can tell these macros should be available here.)